### DISCUSSION TITLE
Cloned phone or sim switched

### DISCUSSION THREAD

**Discussion by ghost** [community/community#138219] [New to GitHub]
I don't know what is going on with this phone. It's a Moto g stylus 5g 2024. The screens changes on me alot. And I have to put in the password twice sometimes 3 times .

**Comment by MasterInQuestion** [comment]
I'd like to know what is going on with your problem.
    Leave more details if you'd like.

**Reply by ghost** [reply]
Thank you for responding. I just bought a USB cable, one end as a C and the other end is HDMI. Only because I've tried every which way possible to sync my devices to mirror my screen. I have a Motorola I have a Samsung and I have an off-brand phone and I have my school's Chromebook that I don't get to use for downloading anything. Also I have my own Wi-Fi network now. Unfortunately I don't have a computer yet. I have an older TV and a xumi box to stream from. Anyways, I could send you many screenshots that just boggle my mind.

**Comment by MasterInQuestion** [comment]
[[
= Cloned phone or SIM switched =

[ Raeford43 @ CE 2024-09-09 21:04:14 UTC:
[https://github.com/orgs/community/discussions/138219#discussion-7162150](https://github.com/orgs/community/discussions/138219#discussion-7162150)
    I don't know what is going on with this phone.
    It's a Motorola Moto G Stylus 5G (2024).

    The screen changes on me alot.
    And I have to put in the password twice sometimes 3 times. ]

&#x2D;---

[ Raeford43 @ CE 2024-09-12 03:12:04 UTC:
[https://github.com/orgs/community/discussions/138219#discussioncomment-10621139](https://github.com/orgs/community/discussions/138219#discussioncomment-10621139)
    Thank you for responding.

    I just bought a USB cable, one end as Type-C and the other end is HDMI.
    Only because I've tried every way which possible: to sync my devices to mirror my screen.

    I have a Motorola, I have a Samsung, and I have an off-brand phone.
    And I have my school's Chromebook that I don't get to use for downloading anything.
    Also I have my own Wi-Fi network now.

    Unfortunately I don't have a computer yet.
    I have an older TV and a Xumo box to stream from.
    Anyways, I could send you many screenshots that just boggle my mind. ]
]]
    So what is your real question..?
    Somehow failed screen sharing (to TV etc.) and perennial auth failure?

    What do you mean by saying "Cloned phone or SIM switched"?

    https://en.wikipedia.org/wiki/SIM_swap_scam
    That's why I distaste phone number based MFA (Multi-Factor Authentication, or 2FA).
    Actually I deem well controlled 1FA (mere password) shall be far more reliable.

    See also:
    https://github.com/orgs/community/discussions?discussions_q=2FA+mandatory
    https://github.com/orgs/community/discussions/68240
    https://bugzilla.mozilla.org/show_bug.cgi?id=1915417#c0
